tests/conda_env/test_create.py

```diff
@@ -250,3 +253,16 @@ def test_create_update_remote_env_file(
         # This ends up sticking around since there is no real way of knowing that an environment
         # variable _used_ to be in the variables dict, but isn't any more.
         assert env_vars["GETS_DELETED"] == "not_actually_removed_though"
+
+
+@pytest.mark.skipif(on_win, reason="Test is invalid on Windows")
+def test_fail_to_create_env_in_dir_with_colon(tmp_path: Path):
+    # Add a directory with a colon
+    colon_dir = tmp_path / "fake:dir"
+    colon_dir.mkdir()
+
+    with pytest.raises(
+        CondaValueError,
+        match="Cannot create a conda environment with ':' in the prefix.",
+    ):
+        run_command(Commands.CREATE, "colon_env", f"--prefix={colon_dir}/tester")
```

Comment:
`run_command` is deprecated, please use `conda_cli` instead:

```suggestion
@pytest.mark.skipif(on_win, reason="Test is invalid on Windows")
def test_fail_to_create_env_in_dir_with_colon(tmp_path: Path, conda_cli: CondaCLIFixture):
    # Add a directory with a colon
    colon_dir = tmp_path / "fake:dir"
    colon_dir.mkdir()

    with pytest.raises(
        CondaValueError,
        match="Cannot create a conda environment with ':' in the prefix.",
    ):
        conda_cli("create", f"--prefix={colon_dir}/tester")
```